growthbook/growthbook
# Update the SQL editor for identifier joins

### Features and Changes


Updated the SQL editor for identifier joins. Now users will be able to test the query before saving.


- Closes **https://github.com/growthbook/growthbook/issues/1769**


### Testing


1. Navigate to Datasources page and add identifier joins.
2. Users must see the new SQL editor.
3. Add a sample query with required columns and test the query.
4. Users must see the sample result of the query stating whether the result is accepted or not.





### Video
https://github.com/growthbook/growthbook/assets/86364607/c024846e-da02-421a-a6ba-bd2b2feb05ce
